app/controllers/sessions_controller.rb

```diff
@@ -20,4 +20,12 @@ def create
       render :new
     end
   end
+
+  def destroy
```

Comment:
Should we add the destroy method to `before_action :redirect_to_home, if: :logged_in?, only: %i[new create]` 🤔

Reply:
I think we shouldn't because `destroy` method used by authenticated user if I add to `before_action` then `destroy` method will not be called from authenticated user.

Reply:
Ahh my bad, you're right @gutakk

Reply:
Fixed in c7b4c98946e036157413fa55101b29c93f4c04c5
What do you think @andyduong1920 @longnd ?